bucket/tightvnc.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "2.8.27",
+    "description": "VNC-Compatible Remote Control / Remote Desktop",
+    "homepage": "http://tightvnc.net",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "32bit": {
+            "url": "https://www.tightvnc.com/download/2.8.27/tightvnc-2.8.27-gpl-setup-32bit.msi",
+            "hash": "e9cb9aa7cfcf2216959184bd3f03447f0685c0b7b0a7352e59a5db3a7e8c1972"
+        },
+        "64bit": {
+            "url": "https://www.tightvnc.com/download/2.8.27/tightvnc-2.8.27-gpl-setup-64bit.msi",
+            "hash": "84132a40da136e61d053bd850d97c74dacf2c47b76dccab5c4f52aaea64a8827"
+        }
+    },
+    "extract_dir": "PFiles\\TightVNC",
+    "bin": [
+        "tvnviewer.exe",
+        "tvnserver.exe"
+    ],
+    "shortcuts": [
+        [
+            "tvnviewer.exe",
+            "TighVNC Viewer"
```

Comment:
```suggestion
            "TightVNC Viewer"
```

Reply:
it is TightVNC isn't it ?

Reply:
What do you think?

![A](https://i.imgur.com/O718oIM.png)

Reply:
indeed I made the mistake :p